tests/textbook_test.py

```diff
@@ -0,0 +1,50 @@
+import unittest
+import timeout_decorator
+
+from PittAPI import textbook
+
```

Comment:
Please with testing use the error and default timeout inside of `__init__.py` instead of `30` and `PittServerDownException`. The only changes that need to accomplish this are to add the import below...

```python
from . import PittServerError, DEFAULT_TIMEOUT
```

and change all the decorators to...

```python
@timeout_decorator.timeout(DEFAULT_TIMEOUT, timeout_exception=PittServerError)
```

This will make the test look cleaner and keep it consistent with all the other tests.